### DISCUSSION TITLE
[BUG] ctrl + v not working properly in VS Code

### DISCUSSION THREAD

**Discussion by Ahorts** [mylinuxforwork/dotfiles#272] [General]
**Describe the bug**
When I try to paste copied text in VS code, nothing happens. The only way I could get it to work is using super + v to open the clipboard, click on the just copied item, before ctrl + v works in VS Code. So far this is the only app I seem to have this issue with is this. Though I noticed this started roughly after the last dotfiles update.

**Expected behavior**
Ctrl + v should just paste the recently copied item into Code.

**Distribution (please complete the following information):**
 - Arch Linux

**Comment by armenr** [comment]
This might relate to a known issue with hyprland, where a regression bug was introduced that partially breaks clipboard sync.

I created this little tool to help work around/fix it --> You can exec-once this in hyprland to have it run in the background.


https://gist.github.com/armenr/81b77587c1dda1d00d1c1c9541dcda94

The script references instances where similar issues are reported by many others. Can you have a look and see if whether this hyprland issue is what's causing you the trouble? I suspect ML4W isn't the culprit, but rather hyprland is.

Snippet from the script that contains references to instances of this issue being reported:

```
# Workaround for issue:
# "Clipboard synchronization between wayland and xwayland clients broken"
# https://github.com/hyprwm/Hyprland/issues/6132
#
# Also pertains to:
# https://github.com/hyprwm/Hyprland/issues/6247
# https://github.com/hyprwm/Hyprland/issues/6443
# https://github.com/hyprwm/Hyprland/issues/6148
```

**Comment by Ahorts** [comment]
> This might relate to a known issue with hyprland, where a regression bug was introduced that partially breaks clipboard sync.
> 
> I created this little tool to help work around/fix it --> You can exec-once this in hyprland to have it run in the background.
> 
> https://gist.github.com/armenr/81b77587c1dda1d00d1c1c9541dcda94
> 
> The script references instances where similar issues are reported by many others. Can you have a look and see if whether this hyprland issue is what's causing you the trouble? I suspect ML4W isn't the culprit, but rather hyprland is.
> 
> Snippet from the script that contains references to instances of this issue being reported:
> 
> ```
> # Workaround for issue:
> # "Clipboard synchronization between wayland and xwayland clients broken"
> # https://github.com/hyprwm/Hyprland/issues/6132
> #
> # Also pertains to:
> # https://github.com/hyprwm/Hyprland/issues/6247
> # https://github.com/hyprwm/Hyprland/issues/6443
> # https://github.com/hyprwm/Hyprland/issues/6148
> ```

Thanks,  but this didn't work for me ☹️. Unless I'm missing something.
Text copied from brave browser does not appear in the system clipboard until pasted elsewhere first. There may be more apps affected.

Output when I run vs code from terminal
```sh

Warning: 'ozone-platform-hint' is not in the list of known options, but still passed to Electron/Chromium.
Warning: 'gtk-version' is not in the list of known options, but still passed to Electron/Chromium.
Warning: 'ignore-gpu-blocklist' is not in the list of known options, but still passed to Electron/Chromium.
Warning: 'enable-features' is not in the list of known options, but still passed to Electron/Chromium.
Warning: 'enable-wayland-ime' is not in the list of known options, but still passed to Electron/Chromium.

```

**Comment by mylinuxforwork** [comment]
@Ahorts I have converted this issue into a discussion because it's not a dotfiles issue.
